src/generator/gen_unicode.ml

```diff
@@ -91,7 +91,8 @@ let files =
               let interval = parse_interval interval in
               let prop = parse_prop prop in
               Hashtbl.add labels prop interval
-          | _ -> assert false );
+          (* TODO: support non-binary properties? *)
+          | _ -> () );
```

Comment:
I think it would be better to stay strict about what we accept. See #158.
That way, we always have to think about what to do whenever new format is introduced.